source/neuropod/bindings/c/c_api_internal.h

```diff
@@ -0,0 +1,23 @@
+/* Copyright (c) 2020 UATC, LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#pragma once
+
+#include "neuropod/neuropod.hh"
+
+struct NP_Neuropod
+{
+    std::unique_ptr<neuropod::Neuropod> model;
```

Comment:
nit: if unique_ptr is used explicitly, it makes sense to include <memory> explicitly too. The same about shared_ptr

Reply:
Fixed